### ISSUE TITLE
Example of input

### ISSUE THREAD

**Issue by sxmon** [naiveHobo/InvoiceNet#1]
Hello,

This project looks interesting. In the paper it says that input is in HOCR format. Can you help me to figure out how to input a document to this project?

Have a good day !

**Comment by naiveHobo** [comment]
Hello,
I skipped the implementation of the preprocessing described in the paper. The model takes the extracted text from a document as the input. The text extraction from a document can be performed using OCRs like tesseract. You need to feed in information such as the position on the page of a word or a phrase that you want to classify, number of characters, etc. Sorry about the  lack of documentation for the project, I'll fix that soon. Hope the code helps you out!

**Comment by bikash** [comment]
Is it possible to share input training data or can you mention the steps to get .pickle file from ocr text

**Comment by dexeotechnologie** [comment]
Hello, I'm really impressed by your work, but I can not understand how the dataframe used to extract the features is built. Can you show an example files / words / coords / labels please?
It would be really useful

Regards

**Comment by sheikmohdimran** [comment]
You can use pytesseract to extract words, and co-ordinates. Unclear on the format of the pickle file or the label assignment though.

https://stackoverflow.com/questions/34248492/how-to-get-hocr-output-using-python-tesseract

**Comment by sheikmohdimran** [comment]
OK from the code, it seems there is atleast 18 labels
self.label_dict = {0: 0, 1: 1, 2: 2, 8: 3, 14: 4, 18: 5}

and these are mapped as 
label_dict = {0: "Other",
              1: "Invoice Date",
              2: "Invoice Number",
              3: "Buyer GST",
              4: "Seller GST",
              5: "Total Amount"}

**Comment by stackoverflowed** [comment]
This looks like a very promising project. Would love to see some input formats though. Would that be possible?

**Comment by divz1010** [comment]
Hi Sarthak,

It will be very helpful if you provide a dataset for this example. I am unable to test/run this script.

**Comment by mon28** [comment]
> OK from the code, it seems there is atleast 18 labels
> self.label_dict = {0: 0, 1: 1, 2: 2, 8: 3, 14: 4, 18: 5}
> 
> and these are mapped as
> label_dict = {0: "Other",
> 1: "Invoice Date",
> 2: "Invoice Number",
> 3: "Buyer GST",
> 4: "Seller GST",
> 5: "Total Amount"}

Hi Imran, were you able to figure out how to label the text?

**Comment by hicka** [comment]
Any updates?

**Comment by naiveHobo** [comment]
I added the data set file and some supporting files in the following commit: https://github.com/naiveHobo/InvoiceNet/commit/a9311967975609ff5e79bbbc01881b107e816953
Closing this issue.

**Comment by oliverbj** [comment]
Sorry for opening this issue again, but it is not clear for me how to use this project.

@sheikmohdimran were you able to figure out how to serve the hOCR output from tesseract to this model?
